app/controllers/v0/mhv_opt_in_flags_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module V0
+  class MHVOptInFlagsController < ApplicationController
+    def show
+      opt_in_flag = MHVOptInFlag.find_by(user_account_id: current_user.user_account, feature: params[:feature])
+      raise Common::Exceptions::RecordNotFound if opt_in_flag.nil?
+
+      render json: { mhv_opt_in_flag: { user_account_id: opt_in_flag.user_account_id, feature: opt_in_flag.feature } }
+    rescue => e
+      render json: { errors: e }, status: :not_found
+    end
+
+    def create
+      feature = params[:feature]
+      unless MHVOptInFlag::FEATURES.include?(feature)
+        raise MHVOptInFlagFeatureNotValid,
+              message: 'Feature param is not valid'
+      end
+
+      status = :ok
+      opt_in_flag = MHVOptInFlag.find_or_create_by(user_account: current_user.user_account,
+                                                   feature: feature) do |_mhv_opt_in_flag|
+        status = :created
+      end
+      render json: { mhv_opt_in_flag: { user_account_id: opt_in_flag.user_account_id, feature: opt_in_flag.feature } },
+             status: status
+    rescue MHVOptInFlagFeatureNotValid => e
+      render json: { errors: e }, status: :bad_request
```

Comment:
Confirmed, with incorrect feature param, get:
`{"errors":"{:message=>\"Feature param is not valid\"}"}`